cli/cmd/sync_v3.go

```diff
@@ -81,6 +73,29 @@ func syncConnectionV3(ctx context.Context, sourceClient *managedplugin.Client, d
 			return err
 		}
 	}
+	// replace @destination.connection with the actual GRPC connection string from the client
+	// NOTE: if this becomes a stable feature, it can move out of sync_v3 and into sync.go
+	replacementValues := make([]specs.ReplacementValue, len(destinationSpecs))
```

Comment:
I think the search and replace should be on string and not on go structs.

Reply:
but maybe not crucial right now as we will are going to change the config anyway